### PR TITLE
Step inputs/outputs/step timeout

### PR DESCRIPTION
**Pull request by zoltan-baba**
- Added an overview on step inputs
- Added info on step outputs and step aliases
- Added a guide on setting step timeout

### CODE REVIEW

File: docs/bitrise-cli/step-inputs.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+```
+- MY_KEY_FOR_THE_ENV: my value for the env
+  opts:
+    title: An example env var item
+    is_dont_change_value: false
+    category: example
+```
+
+- `MY_KEY_FOR_THE_ENV`: the key of the environment item (required)
```
Comment (BirmacherAkos) [inline docs/bitrise-cli/step-inputs.md]: It should be written lowercase like:
my_key_for_the_env

File: docs/bitrise-cli/step-inputs.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+### Using template expressions for step inputs
+
+If you need a step to use a certain value only in certain circumstances, use template expressions as step inputs. Template expressions are evaluated before the step uses the input. They are written in Go's template language - you can read more about that [here](https://golang.org/pkg/text/template/).
+
+Set the `is_template` attribute in the `bitrise.yml` file of your project to use template expressions.
+
+1. Open the `bitrise.yml` file of your project.
+
+1. Find the step in which you wish to use a template expression.
```
Comment (BirmacherAkos) [inline docs/bitrise-cli/step-inputs.md]: `Find the step`.
Not `find the input`?

**Comment by BirmacherAkos** [comment]
Please check my comments

File: docs/bitrise-cli/step-inputs.md (hunk 1 above)
Comment (zoltan-baba) [reply docs/bitrise-cli/step-inputs.md]: Done, thanks!

File: docs/bitrise-cli/step-inputs.md (hunk 2 above)
Comment (zoltan-baba) [reply docs/bitrise-cli/step-inputs.md]: Well, preferably both :) First the step and then the input of that step. I've clarified.

**Pull request merged**